recipe/conda_build_config.yaml

```diff
@@ -7,7 +7,6 @@ blas_impl:
 #    For example, we need a win-64 vs2008_win-32 package, so that we compile win-32
 #    code on win-64 miniconda.
 cross_compiler_target_platform:  # [win]
-  - win-32                     # [win]
```

Comment:
this one is good to delete, but mostly not necessary.  This one is only used when creating the compiler activation packages, so it would not affect much to leave it in.

Not that you should leave it in - just explaining why it's different.

Reply:
Dev dooooooocs! :stuck_out_tongue_winking_eye: